### DISCUSSION TITLE
limit speed changing mods to intervals of x0.05

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#17516] [Ideas]
i don't think there is much of a reason to go in increments less than x0.05 since the resulting map would be very similar to the original at that point

x0.05 was chosen so that the resulting limit would still allow traditional dt to be played, x0.05 rate increases are also common in loved/pending/wip/graveyard mania maps(along with x0.1 increases)

limiting intervals also help combat x1.01 dt abuse(or similar) for scoring

affects: half time(HT), daycore(DC), double time(DT), nightcore(NC), wind up(WU), wind down(WD), adaptive speed(AS)

**Comment by TheFantasticLoki** [comment]
I'm not a fan of this idea as I use the customize portion of both dt and half time to play maps at specific bpms for training and practice.
Rather instead of limiting this functionality, the score multiplier should be based on the speed change itself rather than a flat multiplier for the mod.
